boms/common-ee/pom.xml

```diff
@@ -3256,6 +3256,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
```

Comment:
Since this is uncommon case, what about a comment that we include it even though the same classes are in org.glassfish:jakarta.faces to allow propagation downstream (eg. to BOMs)?